services/networking/vpc/features.yaml

```diff
@@ -0,0 +1,38 @@
+common_features:
+  - CCC.F01  # Encryption in Transit Enabled by Default
+  - CCC.F03  # Access/Activity Logs
+  - CCC.F06  # Identity Based Access Control
+  - CCC.F08  # Multi-zone Deployment
+  - CCC.F09  # Monitoring
+  - CCC.F10  # Logging
```

Comment:
Wonder weather we should capture it under a specific feature under VPC called "Flow Logs". Flow logs is a watcher on traffic that flows through the VPC. It doesn't log about events of VPC like creation of subnets, routing tables.